src/site/filters/liquid.js

```diff
@@ -10,6 +10,8 @@ module.exports = function registerFilters() {
   liquid.filters.humanizeTimestamp = dt =>
     moment.unix(dt).format('MMMM D, YYYY');
 
+  liquid.filters.formatDate = (dt, format) => moment(dt).format(format);
```

Comment:
i needed to add this because we do not have a date filter that takes any date string that is not 'YYYY-MM-DD' and a format - we're usually dealing with unix timestamps or a date string that is 'YYYY-MM-DD'.